hcl/hcl.go

```diff
@@ -514,6 +532,27 @@ func CopyBody(target *hclwrite.Body, src *hclsyntax.Body, evalctx *eval.Context)
 	return nil
 }
 
+func getUnknownScopeTraversal(expr hclsyntax.Expression, evalctx *eval.Context) (*hclsyntax.ScopeTraversalExpr, bool) {
+	logger := log.With().
+		Str("action", "parseUnknownTraversal").
```

Comment:
non-blocking: I believe it'd easier to log the exact name of the function as you're already doing in other cases 

```suggestion
		Str("action", "getUnknownScopeTraversal").
```

Reply:
You are right, this happened because when I refactor using gorename it doesn't fix things like the log entries :laughing: . Thanks for catching this.

Reply:
We are not consistent about it, but I like to add <package>.<func> for functions, since just the function is easier to get clashes (different packages, same function names).